### DISCUSSION TITLE
Need a help to understand the IKE_UP behaviour

### DISCUSSION THREAD

**Discussion by hbhokare** [strongswan/strongswan#2250] [Q&A]
We are using Davici strongswan APIs in one of our applications to create a secured tunnel between two IP endpoints. We are seeing continuous IKE_UP events without corresponding IKE_DOWN event on one of the sites where we have not triggered IKE_INIT. Is it expected behavior ?

**Comment by tobiasbrunner** [comment]
No idea. Please provide more details (logs, config, version etc.).
